app/controllers/response_templates_controller.rb

```diff
@@ -7,17 +7,15 @@ class ResponseTemplatesController < ApplicationController
   ADMIN_TYPES = %w[email_reply abuse_report_email_reply].freeze
 
   def index
-    raise ArgumentError, "Missing param type_of" if params[:type_of].blank?
-
-    user_id = params[:type_of] == "personal_comment" ? current_user.id : nil
-    @response_templates = ResponseTemplate.where(type_of: params[:type_of], user_id: user_id)
+    @response_templates = policy_scope(ResponseTemplate).group_by(&:type_of)
```

Comment:
[Concern] This pattern of reusing variable names for different is a new one for me! You did call this out in your PR description, but having `@response_templates` hold different types of data might hurt readability going forward.

[Suggestion] Consider adding a comment with the reasoning here

```suggestion
// This potentially complicates the back-end response (sometimes an array, sometimes a hash), 
// but it appears that this back-end is possibly also serving response templates for other functions 
// (Email & Abuse Report Email) so those requests should continue to function as they currently do.
    @response_templates = policy_scope(ResponseTemplate).group_by(&:type_of)
```